src/preprocessors/preprocessor.ts

```diff
@@ -7,6 +7,16 @@ import { getCodeFromAst } from '../utils/get-code-from-ast';
 import { getSortedNodes } from '../utils/get-sorted-nodes';
 import { examineAndNormalizePluginOptions } from '../utils/normalize-plugin-options';
 
+const colors = {
+    red: '\x1b[31m',
+    yellow: '\x1b[33m'
+} as const;
+
+function colorText(text: string, color: keyof typeof colors): string {
+    const resetColor = '\x1b[0m';
+    return `${colors[color]}${text}${resetColor}`
```

Comment:
We need to detect if the terminal/process/prettier has colors enabled before posting with colors, so that if they're piping this, we don't bake in these terminal codes for an environment that doesn't support it.

Reply:
Went ahead and removed the colors.  Doesn't seem worth adding another library (or adding all the required code locally) for that.